src/components/PickerDay.vue

```diff
@@ -36,6 +36,10 @@ export default {
     pageTimestamp: Number,
     fullMonthName: Boolean,
     allowedToShowView: Function,
+    dayCustomClasses: {
+      type: Function,
+      default: day => { return {} }
```

Comment:
do we need the return {}? i.e maybe just `default: day => {}`?

Reply:
Good catch, will update (was still getting to know Vue when I wrote this)